imports/startup/server/index.js

```diff
@@ -0,0 +1,27 @@
+import './fixtures';
+import './register-api';
+
+Meteor.users.allow({
+  insert(userId, doc){
+    return true;
+  },
+  update(userId, doc, fieldNames, modifer){
+    return true;
+    if(modifier){
+    }
```

Comment:
Remove dead code. Please please get in the habit of maintaining a clean codebase. No projects in your GitHub repos should contain commented code or dead code like line 10 - 11 here!